extras/src/multisig.rs

```diff
@@ -0,0 +1,100 @@
+use clock::ChainEpoch;
+use forest_address::Address;
+use forest_encoding::tuple::*;
+use forest_vm::{MethodNum, Serialized, TokenAmount, METHOD_CONSTRUCTOR};
+use num_bigint::biguint_ser;
+use serde::{Deserialize, Serialize};
+
+/// Transaction ID type
+// TODO change to uvarint encoding
```

Comment:
Is this old of should we create an issue for this?

Reply:
This come from here but I don't know what it means really : https://github.com/Zondax/forest/blob/master/vm/actor/src/builtin/multisig/types.rs#L12-L15

Reply:
https://github.com/Zondax/filecoin-signing-tools/issues/270